src/api/cpp/cvc5.h

```diff
@@ -584,19 +584,25 @@ class CVC5_EXPORT Sort
   bool isUninterpretedSort() const;
 
   /**
-   * Is this a sort constructor kind?
+   * Is this an uninterpreted sort constructor kind?
+   *
+   * An uninterpreted sort constructor is an uninterpreted sort with arity > 0.
```

Comment:
I would not call it an uninterpreted sort.  Note that calling instantiate on it generates sorts that are essentially uninterpreted sorts.

This should perhaps say that it was constructed via `declareSort` with arity > 0.